From afc3f8b7c366441c59c664d0a86acc7e1bfb125c Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Wed, 26 Jul 2017 19:31:53 -0400
Subject: [PATCH 2764/2831] drm/amd/display: Rename firmware_info to
 dc_firmware_info

This is to avoid conflicts with amdgpu's firmware_info once we
merge amdgpu_dm_types with amdgpu_dm.

Change-Id: I0b33ffcf0ab7885e61eaae9aaec1f4239f2a3e92
Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Reviewed-by: Tony Cheng <Tony.Cheng@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
---
 drivers/gpu/drm/amd/display/dc/bios/bios_parser.c          | 14 +++++++-------
 drivers/gpu/drm/amd/display/dc/bios/bios_parser2.c         |  8 ++++----
 drivers/gpu/drm/amd/display/dc/core/dc_resource.c          |  2 +-
 drivers/gpu/drm/amd/display/dc/dc_bios_types.h             |  2 +-
 drivers/gpu/drm/amd/display/dc/dce/dce_clock_source.c      |  4 ++--
 drivers/gpu/drm/amd/display/dc/dce/dce_clocks.c            |  2 +-
 drivers/gpu/drm/amd/display/dc/dce100/dce100_resource.c    |  2 +-
 drivers/gpu/drm/amd/display/dc/dce110/dce110_resource.c    |  2 +-
 drivers/gpu/drm/amd/display/dc/dce80/dce80_resource.c      |  2 +-
 drivers/gpu/drm/amd/display/dc/i2caux/i2caux.c             |  2 +-
 .../gpu/drm/amd/display/include/grph_object_ctrl_defs.h    |  2 +-
 11 files changed, 21 insertions(+), 21 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/bios/bios_parser.c b/drivers/gpu/drm/amd/display/dc/bios/bios_parser.c
index 47a4676..2c683d4 100644
--- a/drivers/gpu/drm/amd/display/dc/bios/bios_parser.c
+++ b/drivers/gpu/drm/amd/display/dc/bios/bios_parser.c
@@ -622,17 +622,17 @@ static enum bp_result bios_parser_get_device_tag(
 
 static enum bp_result get_firmware_info_v1_4(
 	struct bios_parser *bp,
-	struct firmware_info *info);
+	struct dc_firmware_info *info);
 static enum bp_result get_firmware_info_v2_1(
 	struct bios_parser *bp,
-	struct firmware_info *info);
+	struct dc_firmware_info *info);
 static enum bp_result get_firmware_info_v2_2(
 	struct bios_parser *bp,
-	struct firmware_info *info);
+	struct dc_firmware_info *info);
 
 static enum bp_result bios_parser_get_firmware_info(
 	struct dc_bios *dcb,
-	struct firmware_info *info)
+	struct dc_firmware_info *info)
 {
 	struct bios_parser *bp = BP_FROM_DCB(dcb);
 	enum bp_result result = BP_RESULT_BADBIOSTABLE;
@@ -676,7 +676,7 @@ static enum bp_result bios_parser_get_firmware_info(
 
 static enum bp_result get_firmware_info_v1_4(
 	struct bios_parser *bp,
-	struct firmware_info *info)
+	struct dc_firmware_info *info)
 {
 	ATOM_FIRMWARE_INFO_V1_4 *firmware_info =
 		GET_IMAGE(ATOM_FIRMWARE_INFO_V1_4,
@@ -726,7 +726,7 @@ static enum bp_result get_ss_info_v3_1(
 
 static enum bp_result get_firmware_info_v2_1(
 	struct bios_parser *bp,
-	struct firmware_info *info)
+	struct dc_firmware_info *info)
 {
 	ATOM_FIRMWARE_INFO_V2_1 *firmwareInfo =
 		GET_IMAGE(ATOM_FIRMWARE_INFO_V2_1, DATA_TABLES(FirmwareInfo));
@@ -810,7 +810,7 @@ static enum bp_result get_firmware_info_v2_1(
 
 static enum bp_result get_firmware_info_v2_2(
 	struct bios_parser *bp,
-	struct firmware_info *info)
+	struct dc_firmware_info *info)
 {
 	ATOM_FIRMWARE_INFO_V2_2 *firmware_info;
 	struct spread_spectrum_info internal_ss;
diff --git a/drivers/gpu/drm/amd/display/dc/bios/bios_parser2.c b/drivers/gpu/drm/amd/display/dc/bios/bios_parser2.c
index 3a69f9c..86fce5a 100644
--- a/drivers/gpu/drm/amd/display/dc/bios/bios_parser2.c
+++ b/drivers/gpu/drm/amd/display/dc/bios/bios_parser2.c
@@ -81,7 +81,7 @@ static enum bp_result get_gpio_i2c_info(struct bios_parser *bp,
 
 static enum bp_result bios_parser_get_firmware_info(
 	struct dc_bios *dcb,
-	struct firmware_info *info);
+	struct dc_firmware_info *info);
 
 static enum bp_result bios_parser_get_encoder_cap_info(
 	struct dc_bios *dcb,
@@ -90,7 +90,7 @@ static enum bp_result bios_parser_get_encoder_cap_info(
 
 static enum bp_result get_firmware_info_v3_1(
 	struct bios_parser *bp,
-	struct firmware_info *info);
+	struct dc_firmware_info *info);
 
 static struct atom_hpd_int_record *get_hpd_record(struct bios_parser *bp,
 		struct atom_display_object_path_v2 *object);
@@ -1583,7 +1583,7 @@ static void bios_parser_set_scratch_critical_state(
 
 static enum bp_result bios_parser_get_firmware_info(
 	struct dc_bios *dcb,
-	struct firmware_info *info)
+	struct dc_firmware_info *info)
 {
 	struct bios_parser *bp = BP_FROM_DCB(dcb);
 	enum bp_result result = BP_RESULT_BADBIOSTABLE;
@@ -1615,7 +1615,7 @@ static enum bp_result bios_parser_get_firmware_info(
 
 static enum bp_result get_firmware_info_v3_1(
 	struct bios_parser *bp,
-	struct firmware_info *info)
+	struct dc_firmware_info *info)
 {
 	struct atom_firmware_info_v3_1 *firmware_info;
 	struct atom_display_controller_info_v4_1 *dce_info = NULL;
diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_resource.c b/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
index a4b80a3..36cd077 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
@@ -126,7 +126,7 @@ struct resource_pool *dc_create_resource_pool(
 		break;
 	}
 	if (res_pool != NULL) {
-		struct firmware_info fw_info = { { 0 } };
+		struct dc_firmware_info fw_info = { { 0 } };
 
 		if (dc->ctx->dc_bios->funcs->get_firmware_info(
 				dc->ctx->dc_bios, &fw_info) == BP_RESULT_OK) {
diff --git a/drivers/gpu/drm/amd/display/dc/dc_bios_types.h b/drivers/gpu/drm/amd/display/dc/dc_bios_types.h
index 6e1291d..273d80a 100644
--- a/drivers/gpu/drm/amd/display/dc/dc_bios_types.h
+++ b/drivers/gpu/drm/amd/display/dc/dc_bios_types.h
@@ -84,7 +84,7 @@ struct dc_vbios_funcs {
 		struct connector_device_tag_info *info);
 	enum bp_result (*get_firmware_info)(
 		struct dc_bios *bios,
-		struct firmware_info *info);
+		struct dc_firmware_info *info);
 	enum bp_result (*get_spread_spectrum_info)(
 		struct dc_bios *bios,
 		enum as_signal_type signal,
diff --git a/drivers/gpu/drm/amd/display/dc/dce/dce_clock_source.c b/drivers/gpu/drm/amd/display/dc/dce/dce_clock_source.c
index 2205491..dd7e1dd 100644
--- a/drivers/gpu/drm/amd/display/dc/dce/dce_clock_source.c
+++ b/drivers/gpu/drm/amd/display/dc/dce/dce_clock_source.c
@@ -1103,7 +1103,7 @@ static bool calc_pll_max_vco_construct(
 			struct calc_pll_clock_source_init_data *init_data)
 {
 	uint32_t i;
-	struct firmware_info fw_info = { { 0 } };
+	struct dc_firmware_info fw_info = { { 0 } };
 	if (calc_pll_cs == NULL ||
 			init_data == NULL ||
 			init_data->bp == NULL)
@@ -1185,7 +1185,7 @@ bool dce110_clk_src_construct(
 	const struct dce110_clk_src_shift *cs_shift,
 	const struct dce110_clk_src_mask *cs_mask)
 {
-	struct firmware_info fw_info = { { 0 } };
+	struct dc_firmware_info fw_info = { { 0 } };
 	struct calc_pll_clock_source_init_data calc_pll_cs_init_data_hdmi;
 	struct calc_pll_clock_source_init_data calc_pll_cs_init_data;
 
diff --git a/drivers/gpu/drm/amd/display/dc/dce/dce_clocks.c b/drivers/gpu/drm/amd/display/dc/dce/dce_clocks.c
index 37d074f..a73228b 100644
--- a/drivers/gpu/drm/amd/display/dc/dce/dce_clocks.c
+++ b/drivers/gpu/drm/amd/display/dc/dce/dce_clocks.c
@@ -424,7 +424,7 @@ static void dce_clock_read_integrated_info(struct dce_disp_clk *clk_dce)
 	struct dc_debug *debug = &clk_dce->base.ctx->dc->debug;
 	struct dc_bios *bp = clk_dce->base.ctx->dc_bios;
 	struct integrated_info info = { { { 0 } } };
-	struct firmware_info fw_info = { { 0 } };
+	struct dc_firmware_info fw_info = { { 0 } };
 	int i;
 
 	if (bp->integrated_info)
diff --git a/drivers/gpu/drm/amd/display/dc/dce100/dce100_resource.c b/drivers/gpu/drm/amd/display/dc/dce100/dce100_resource.c
index cf98f247..9ac246f 100644
--- a/drivers/gpu/drm/amd/display/dc/dce100/dce100_resource.c
+++ b/drivers/gpu/drm/amd/display/dc/dce100/dce100_resource.c
@@ -819,7 +819,7 @@ static bool construct(
 {
 	unsigned int i;
 	struct dc_context *ctx = dc->ctx;
-	struct firmware_info info;
+	struct dc_firmware_info info;
 	struct dc_bios *bp;
 	struct dm_pp_static_clock_info static_clk_info = {0};
 
diff --git a/drivers/gpu/drm/amd/display/dc/dce110/dce110_resource.c b/drivers/gpu/drm/amd/display/dc/dce110/dce110_resource.c
index cc25c4b..5c1790b 100644
--- a/drivers/gpu/drm/amd/display/dc/dce110/dce110_resource.c
+++ b/drivers/gpu/drm/amd/display/dc/dce110/dce110_resource.c
@@ -1190,7 +1190,7 @@ static bool construct(
 {
 	unsigned int i;
 	struct dc_context *ctx = dc->ctx;
-	struct firmware_info info;
+	struct dc_firmware_info info;
 	struct dc_bios *bp;
 	struct dm_pp_static_clock_info static_clk_info = {0};
 
diff --git a/drivers/gpu/drm/amd/display/dc/dce80/dce80_resource.c b/drivers/gpu/drm/amd/display/dc/dce80/dce80_resource.c
index 2fde43c..92f01ed 100644
--- a/drivers/gpu/drm/amd/display/dc/dce80/dce80_resource.c
+++ b/drivers/gpu/drm/amd/display/dc/dce80/dce80_resource.c
@@ -833,7 +833,7 @@ static bool construct(
 {
 	unsigned int i;
 	struct dc_context *ctx = dc->ctx;
-	struct firmware_info info;
+	struct dc_firmware_info info;
 	struct dc_bios *bp;
 	struct dm_pp_static_clock_info static_clk_info = {0};
 
diff --git a/drivers/gpu/drm/amd/display/dc/i2caux/i2caux.c b/drivers/gpu/drm/amd/display/dc/i2caux/i2caux.c
index 10061cc..da08cc1 100644
--- a/drivers/gpu/drm/amd/display/dc/i2caux/i2caux.c
+++ b/drivers/gpu/drm/amd/display/dc/i2caux/i2caux.c
@@ -320,7 +320,7 @@ void dal_i2caux_destroy(
 uint32_t dal_i2caux_get_reference_clock(
 		struct dc_bios *bios)
 {
-	struct firmware_info info = { { 0 } };
+	struct dc_firmware_info info = { { 0 } };
 
 	if (bios->funcs->get_firmware_info(bios, &info) != BP_RESULT_OK)
 		return 0;
diff --git a/drivers/gpu/drm/amd/display/include/grph_object_ctrl_defs.h b/drivers/gpu/drm/amd/display/include/grph_object_ctrl_defs.h
index 9c0bf65..6cdbf84 100644
--- a/drivers/gpu/drm/amd/display/include/grph_object_ctrl_defs.h
+++ b/drivers/gpu/drm/amd/display/include/grph_object_ctrl_defs.h
@@ -154,7 +154,7 @@ struct embedded_panel_info {
 	bool realtek_eDPToLVDS;
 };
 
-struct firmware_info {
+struct dc_firmware_info {
 	struct pll_info {
 		uint32_t crystal_frequency; /* in KHz */
 		uint32_t min_input_pxl_clk_pll_frequency; /* in KHz */
-- 
2.7.4

